.github/cspell.json

```diff
@@ -3,6 +3,154 @@
   "language": "en",
   "flagWords": [],
   "words": [
+    "unfollow",
+    "pubdev",
+    "unmount",
+    "Tappable",
+    "tappable",
+    "AHSL",    
+    "Gamepads",
+    "gamepads",
+    "Rgba",
+    "ints",
+    "Skia",
+    "skia",
+    "NPCs",
+    "WBMP",
+    "bloodlust",
+    "BGUG",
+    "subfolders",
+    "prepartion",
+    "updatesystem",
+    "netcode",
+    "Nakama",
+    "perfoming",
+    "gameplay",
+    "spritecow",
+    "mutators",
+    "greyscale",
+    "gameover",
+    "Lukas",
+    "Klingsbo",
+    "highscore",
+    "asprite",
+    "colourful",
+    "polyline",
+    "Coord",
+    "Raycasting",
+    "Bezier",
+    "Collidable"
+    "collidable",
+    "draggables",
+    "draggable",
+    "Padracing",
+    "padracing",
+    "trex",
+    "Draggable",
+    "Hoverable",
+    "Draggables",
+    "Hoverables",
+    "WASD",
+    "Multitap",
+    "fullscreen",
+    "tweening",
+    "abelian",
+    "Tappables",
+    "SVGs",
+    "svgs",
+    "ballons",
+    "Ballons",
+    "respawned",
+    "respawn",
+    "Respawned",
+    "Kinect",
+    "everytime",
+    "caveace",
+    "ptero",
+    "pubspec",
+    "Datas",
+    "Kawabunga",
+    "artboard",
+    "Artboard",
+    "antialiasing",
+    "goldens",
+    "erayzesen",
+    "Rects",
+    "Tiled's",
+    "unrenderable",
+    "tilesets",
+    "feroult",
+    "erickzanardo",
+    "Schnurber",
+    "Akida",
+    "pgainullin",
+    "luanpotter",
+    "lexer's",
+    "serializable",
+    "webp",
+    "wbmp",
+    "reimplement",
+    "raycasts",
+    "collidables",
+    "Queryable",
+    "overridable",
+    "reimplemented",
+    "Bitfield",
+    "preorder",
+    "rebalance",
+    "tilemap",
+    "tileset",
+    "Hitboxes",
+    "hitboxes",
+    "Widget's",
+    "spydon",
+    "Fullscreen",
+    "Mutators",
+    "Deletable",
+    "Foo's",
+    "mixins",
+    "Viewport's",
+    "viewport's",
+    "viewports",
+    "janky",
+    "behaviour",
+    "unproject",
+    "ambiguate",
+    "microtask",
+    "autofocus",
+    "Draggable",
+    "Tappable",
+    "postmultiply",
+    "euclidian",
+    "raycasting",
+    "Tavian",
+    "cathetus",
+    "unscale",
+    "Composability",
+    "wolfenrain",
+    "lerp",
+    "RGBO",
+    "ARGB",
+    "grayscale",
+    "Aseprite",
+    "ssin",
+    "scos",
+    "Ints",
+    "glyph's",
+    "performant",
+    "codepoint",
+    "codepoints",
+    "rrect",
+    "rects",
+    "rsts",
+    "LTRB",
+    "sublist",
+    "monospace",
+    "Arial",
+    "spritesheet",
+    "gapless",
+    "Audioplayers",
+    "Patreon",
```

Comment:
This list should be sorted, and words starting with a capital letter shouldn't need to be added I think (they should be treated as names).

Reply:
thanks @spydon check if there is something more to do i'll fix it asap